src/main/java/com/akkus/gen/VocidService.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.akkus.gen;
+
+import java.util.logging.Logger;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+import org.springframework.stereotype.Service;
+
+/**
+ *
+ * @author Abdurrahman AKKUŞ
+ */
+@Service
+public class VocidService extends ISuperService<ISuperEntity> {
```

Comment:
I would've liked a clearer seperation between the repository and service layers.